Firestore/Example/Tests/Integration/API/FIRCursorTests.mm

```diff
@@ -192,4 +192,131 @@ - (void)testCanBeUsedInDescendingQueries {
   XCTAssertEqualObjects(FIRQuerySnapshotGetData(snapshot), (@[ @{ @"v" : @"d", @"sort" : @3.0 } ]));
 }
 
+- (void)testTimestampsCanBePassedToQueriesAsLimits {
+  // Firestore only supports microsecond resolution, so use a microsecond as a minimum value for
+  // nanoseconds.
+  int32_t us = 1000;
+  FIRCollectionReference *testCollection = [self collectionRefWithDocuments:@{
+    @"a" :
+        @{@"k" : @"a", @"timestamp" : [FIRTimestamp timestampWithSeconds:100 nanoseconds:1 * us]},
+    @"b" :
+        @{@"k" : @"b", @"timestamp" : [FIRTimestamp timestampWithSeconds:100 nanoseconds:2 * us]},
+    @"c" :
+        @{@"k" : @"c", @"timestamp" : [FIRTimestamp timestampWithSeconds:100 nanoseconds:3 * us]},
```

Comment:
This test would succeed even if you remove the orderedByField because of the default order by key. Please scatter the timestamp values amongst the keys to show that the result is not the same order as the keys.

Reply:
Done (in the next test as well).